src/gax-internal/src/http.rs

```diff
@@ -0,0 +1,443 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use auth::credentials::{create_access_token_credential, Credential};
+use gax::backoff_policy::BackoffPolicy;
+use gax::error::Error;
+use gax::error::HttpError;
+use gax::error::ServiceError;
+use gax::exponential_backoff::ExponentialBackoff;
+use gax::loop_state::LoopState;
+use gax::polling_backoff_policy::PollingBackoffPolicy;
+use gax::polling_policy::Aip194Strict;
+use gax::polling_policy::PollingPolicy;
+use gax::retry_policy::RetryPolicy;
+use gax::retry_throttler::SharedRetryThrottler;
+use gax::Result;
+use std::sync::Arc;
+
+#[doc(hidden)]
```

Comment:
comment: when everything is behind a feature, we will not need these `#[doc(hidden)]` tags anymore

Reply:
Ack.